tests/test_converter.py

```diff
@@ -381,6 +381,23 @@ class C:
     assert inst == converter.structure(unstructured, C)
 
 
+def test_dict_roundtrip_with_alias():
+    """
+    A class with an aliased attribute can be unstructured and structured.
+    """
+
+    converter = Converter()
```

Comment:
I think you should be testing this with a `BaseConverter` to hit the changed code path, the `Converter` uses different machinery.